.github/workflows/build.yml

```diff
@@ -63,7 +63,18 @@ jobs:
         run: |
           sudo apt update
           sudo apt install -y --no-install-recommends \
-            ninja-build cmake g++ libsdl1.2-dev libsdl-image1.2-dev libncurses-dev zlib1g-dev
+            ninja-build cmake g++ libsdl1.2-dev libsdl-image1.2-dev libncurses-dev zlib1g-dev libspdlog-dev
+
+      - name: Set up vcpkg on windows
+        if: ${{ matrix.os.preset == 'win' }}
+        run: |
+          git clone https://github.com/Microsoft/vcpkg.git
+          .\vcpkg\bootstrap-vcpkg.bat
+
+      - name: Install dependencies with vcpkg
+        if: ${{ matrix.os.preset == 'win' }}
+        run: |
+          .\vcpkg\vcpkg install spdlog
```

Comment:
You don't need that in pipeline. Just add spdlog to vcpkg.json

Reply:
yeah sorry, should be all set now, all checks pass.